postgresql-containers/bin/postgres-readiness-check.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
maybe we can remove this file at all and use our custom only for liveness @egegunes @egegunes WDYT?

Reply:
actually makes sense

Reply:
we discussed with the team, we will keep it for now given that we want in the future to have the option to defined the probes through configmaps